config/develop/bridge-exporter.yaml

```diff
@@ -11,11 +11,14 @@ parameters:
   BridgeWorkerPassword: !ssm /bridge-exporter-develop/BridgeWorkerPassword
   EC2InstanceType: t2.micro
   Env: develop
+  ExporterSqsQueueUrl: https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-EX-Request-develop
   NewRelicAppName: bridge-exporter-develop
   NewRelicLicenseKey: !ssm /infra/NewRelicLicenseKey
-  RecordIdsBucket: org-sagebridge-exporter-recordids-develop
+  RecordIdsBucket: org-sagebridge-exporter-recordids-devdevelop
+  S3NotificationSqsQueueUrl: https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-UploadComplete-Notification-develop
```

Comment:
It's ok this is "develop" and not "devdevelop"? (Same with staging below)

Reply:
Yes, this is intentional. S3 bucket names are global, not namespaced to your account, so I needed to change the bucket name for the AWS dev account.